cypress/support/customCommands/user/index.js

```diff
@@ -47,3 +47,9 @@ Cypress.Commands.add(
     });
   },
 );
+
+Cypress.Commands.add("loginUserViaMockedToken", () => {
+  cy.window().then(appWindow => {
+    appWindow.localStorage.setItem("_saleorCSRFToken", "#token#");
```

Comment:
question: should we have one mock token with some random data inside? Right now to run test required pass own token there, but this will be problem in CI, so IMO better will be use mocked token.

Reply:
@karolm-saleor @poulch . So for now, I have to log in manually, then copy the token and paste it here. Won't it be better to just use the existing Command - loginUserViaRequest? This command sets _saleorCSRFToken in browser. Or we could use the [session](https://docs.cypress.io/api/commands/session), with login via API, and cache it across all tests. I don't see value in tests where you need to copy token

Reply:
@andrzejewsky

Reply:
@karola312 we want to mock all requests. We don't need any unique token, the only requirement is that token contains proper structure but the data doesn't have to be accurate, it means such a user may not exists and it's ok.